src/solver/conv_winoRxS.cpp

```diff
@@ -656,7 +679,15 @@ static bool IsApplicableBase(const ExecutionContext& ctx, const ProblemDescripti
         && problem.GetBias() == 0
         && problem.GetInLayout() == "NCHW"))
         return false;
-    // clang-format on
+        // clang-format on
+
+#if WORKAROUND_ISSUE_2493
+    if(!miopen::IsDisabled(MIOPEN_DEBUG_WORKAROUND_ISSUE_2493{}))
+    {
+        if(ShaderModel(ctx, problem, Winodata, Winofilter).GetGranularityLoss() > 0.995)
```

Comment:
Perhaps we can move the constant to a define or a `constexpr` variable so that the limit is documented ?

Reply:
Resolved in PR #2510, commit f186fafede484eff08277fa2392413720df3916c.